src/ucc/api/ucc.h

```diff
@@ -1333,25 +1333,38 @@ ucc_status_t ucc_team_get_all_eps(ucc_team_h team, uint64_t **ep,
  *  @ingroup UCC_COLLECTIVES_DT
  */
 typedef enum {
-    UCC_COLL_BUFF_FLAG_IN_PLACE             = UCC_BIT(0),
-    UCC_COLL_BUFF_FLAG_PERSISTENT           = UCC_BIT(1),
-    UCC_COLL_BUFF_FLAG_COUNT_64BIT          = UCC_BIT(2),
-    UCC_COLL_BUFF_FLAG_DISPLACEMENTS_64BIT  = UCC_BIT(3)
-} ucc_coll_buffer_flags_t;
+    UCC_COLL_ARGS_FLAG_IN_PLACE             = UCC_BIT(0),
+    UCC_COLL_ARGS_FLAG_PERSISTENT           = UCC_BIT(1),
+    UCC_COLL_ARGS_FLAG_COUNT_64BIT          = UCC_BIT(2),
```

Comment:
what is the use of this flag?

Reply:
This flag specifies that the displacements and counts arrays in the invo_v are 8 bytes values in contrast to 4b if flag is not set. @manjugv correct me plz if i'm wrong

Reply:
Correct. It is to support Big Count in MPI.

Reply:
why can't ucc always say it support 64 bit.

Reply:
Remember this impacts both counts and displacements. For vector collectives, given the common case is 32 bit count and displacement, we might have more bytes than necessary (32 bits * number of ranks). We can have a union to combine 32 and 64 bit, but then it is cumbersome.

Reply:
@bureddy main reason, we don't want to force user to re-allocate uint64_t array and re-pack (potentially large) arrays of displs/counts. So we have this flag for the user to say what is the type of storage.